CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+#### Version 7.0.0
+* [BREAKING] Removed deprecated `optOut` method from Thunderhead SDK.
```

Comment:
Not sure if we need version 7.0.0 in Changelog, but think it is ok for the test repo.